test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -165,6 +165,7 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'xpack.cloud.deployment_url (string)',
         'xpack.cloud.full_story.enabled (boolean)',
         'xpack.cloud.full_story.org_id (any)',
+        // No PII. Just the list of event types we want to forward to FullStory.
```

Comment:
These strings are not mandated by FullStory. This config is used to filter the events from EBT that we actually send to FullStory. It has a very aggressive rate limiting policy regarding custom events (up to 10 events per second and 30 per minute), so we want to send only the events from Kibana that we really want to have in FullStory. All the events will still go to our Remote Telemetry Service, it's just a filter to the FullStory endpoint.

Right now it's just `['Loaded Kibana']` because it's the only event we used to send to FullStory from the start. It may grow to match the name of the events we want to forward.

Reply:
Got it, thanks for the detailed explanation.